src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -51,7 +51,6 @@
     <file src="Microsoft.DotNet.Build.Tasks.Packaging.Desktop\Microsoft.DotNet.Build.Tasks.Packaging.dll" target="lib\net46" />
     <file src="Microsoft.DotNet.Build.Tasks.Packaging.Desktop\NuGet.*.dll" target="lib\net46" />
     <file src="Microsoft.DotNet.VersionTools\Microsoft.DotNet.VersionTools.dll" target="lib" />
-    <file src="Microsoft.DotNet.VersionTools.net45\Microsoft.DotNet.VersionTools.dll" target="lib\net46" />
```

Comment:
What's the story here?

Reply:
Looking at it deeper, it shouldn't be necessary. Reverting it and the `<Private>False</Private>` removal. All I really needed to fix the XML DLL load error was the `$(VersionToolsProjectReferenceSuffix)`.

It's originally from https://github.com/dotnet/buildtools/pull/1935. I wanted to be able to load the built build task DLL during the official build to use it to publish to dotnet/versions, but it wouldn't load because the VersionTools DLL couldn't be found. I removed `<Private>false</Private>` so VersionTools would be in the same dir and the load worked. Then this nuspec line was doing a duplicate copy so I removed it.